src/Event/Run/AfterRunSubscriber.php

```diff
@@ -0,0 +1,17 @@
+<?php declare(strict_types=1);
+/*
+ * This file is part of PHPUnit.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace PHPUnit\Event\Run;
+
+use PHPUnit\Event\Subscriber;
+
+interface AfterRunSubscriber extends Subscriber
+{
+    public function notify(AfterRun $event): void;
```

Comment:
Something that would be nice:

```diff
diff --git a/src/Event/Subscriber.php b/src/Event/Subscriber.php
index 4b2640ec4..3c9b50e70 100644
--- a/src/Event/Subscriber.php
+++ b/src/Event/Subscriber.php
@@ -11,4 +11,5 @@

 interface Subscriber
 {
+    public function notify(Event $event);
 }
```

Reply:
Target PHP 7.4 and go for it :smile: 

https://wiki.php.net/rfc/covariant-returns-and-contravariant-parameters